---
title: "& č. 39; Ukončit zkuste & č. 39; může být použit pouze uvnitř a & č. 39; Zkuste & č. 39; příkaz"
ms.date: 07/20/2015
ms.prod: .net
ms.technology: devlang-visual-basic
ms.topic: article
f1_keywords:
- vbc30393
- bc30393
helpviewer_keywords: BC30393
ms.assetid: b8651df3-a32f-478c-a6d8-aa0ef584155f
caps.latest.revision: "7"
author: dotnet-bot
ms.author: dotnetcontent
ms.openlocfilehash: 0eaf43bf42d1a323221d79dabf2bff531abcfe6f
ms.sourcegitcommit: 4f3fef493080a43e70e951223894768d36ce430a
ms.translationtype: MT
ms.contentlocale: cs-CZ
ms.lasthandoff: 11/21/2017
---
# <a name="39exit-try39-can-only-appear-inside-a-39try39-statement"></a>& č. 39; Ukončit zkuste & č. 39; může být použit pouze uvnitř a & č. 39; Zkuste & č. 39; příkaz
`Exit Try`lze použít pouze v rámci `Try` blokovat příkaz. Máte buď redundantní `Exit Try` prohlášení, nebo vaše `Exit Try` příkaz zobrazí mimo hranice jeho odpovídající `Try` bloku.  
  
 **ID chyby:** BC30393  
  
## <a name="to-correct-this-error"></a>Oprava této chyby  
  
1.  Vyhledejte a odeberte nadbytečné `Exit Try` příkaz.  
  
2.  Přesunout `Exit Try` příkaz na příslušné místo vašeho kódu.  
  
## <a name="see-also"></a>Viz také  
 [Try... Catch... Finally – příkaz](../../visual-basic/language-reference/statements/try-catch-finally-statement.md)  
 [Přehled strukturovaného zpracování výjimek v jazyce Visual Basic](http://msdn.microsoft.com/en-us/bb81af80-a735-4873-9711-6151a48e418a)
